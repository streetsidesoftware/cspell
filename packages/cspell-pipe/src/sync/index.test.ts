import * as pipe from '.';

describe('Pipe Sync API', () => {
    test('pipe api', () => {
        expect(Object.keys(pipe).sort()).toMatchSnapshot();
    });
});
