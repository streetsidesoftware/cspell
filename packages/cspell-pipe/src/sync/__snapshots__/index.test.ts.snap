// Vitest Snapshot v1

exports[`Pipe Sync API > pipe api 1`] = `
[
  "opAppend",
  "opCombine",
  "opConcatMap",
  "opFilter",
  "opFirst",
  "opFlatten",
  "opJoinStrings",
  "opLast",
  "opMap",
  "opReduce",
  "opSkip",
  "opTake",
  "opTap",
  "opUnique",
  "pipe",
  "pipeSync",
  "reduce",
  "toArray",
]
`;
